packages/fluent-theme/src/components/sendbox/Sendbox.tsx

```diff
@@ -0,0 +1,172 @@
+  const [files, setFiles] = useState<File[]>([]);
+  const isMessageLengthExceeded = !!props.maxMessageLength && message.length > props.maxMessageLength;
+  const classnames = useStyles(styles);
+  const localize = useLocalizer();
+
+  const handleReplyClick = useCallback((action: Partial<DirectLineCardAction>) => {
+    if ('displayText' in action) {
+      props.onPostMessage?.({ text: action.displayText });
+    }
+  }, []);
+
+  const handleSendboxClick = useCallback<MouseEventHandler>(event => {
+    if ('tabIndex' in event.target && typeof event.target.tabIndex === 'number' && event.target.tabIndex >= 0) {
+      return;
+    }
+    inputRef.current?.focus();
+  }, []);
+
+  const handleMessageChange: React.FormEventHandler<HTMLTextAreaElement> = useCallback(ev => {
+    setMessage(ev.currentTarget.value);
+  }, []);
+
+  const handleAddFiles = useCallback((inputFiles: File[]) => {
+    setFiles(files => files.concat(inputFiles));
+  }, []);
+
+  const handleSendClick = useCallback(() => {
+    props.onPostMessage?.({
+      text: message,
+      attachments: files.map(file => ({
+        contentType: file.type,
+        name: file.name
+      }))
+    });
+    setMessage('');
+    setFiles([]);
+  }, [props, message, files]);
+
+  const aria = {
+    role: 'form',
+    'aria-invalid': 'false' as const,
+    ...(props.errorMessageId && {
+      'aria-invalid': 'true' as const,
+      'aria-errormessage': props.errorMessageId
+    })
+  };
+
+  return (
+    <div className={cx(classnames.webchat__sendbox__root, props.className)} {...aria}>
+      {props.suggestedActions && (
+        <SuggestedActions onActionClick={handleReplyClick} suggestedActions={props.suggestedActions} />
+      )}
+      <div className={cx(classnames.webchat__sendbox__sendbox)} onClickCapture={handleSendboxClick}>
+        <TextArea
+          className={cx(classnames['webchat__sendbox__sendbox-text'])}
+          onInput={handleMessageChange}
+          placeholder={props.placeholder ?? localize('TEXT_INPUT_PLACEHOLDER')}
+          ref={inputRef}
+          value={message}
+        />
+        <Attachments files={files} />
+        <div className={cx(classnames['webchat__sendbox__sendbox-controls'])}>
+          {props.maxMessageLength && (
+            <div
+              className={cx(classnames['webchat__sendbox__text-counter'], {
+                [classnames['webchat__sendbox__text-counter--error']]: isMessageLengthExceeded
+              })}
+            >
+              {`${message.length}/${props.maxMessageLength}`}
+            </div>
+          )}
+          <Toolbar>
+            <ToolbarButton aria-label={localize('TEXT_INPUT_DIALPAD_BUTTON_ALT')}>
+              <DialpadIcon />
+            </ToolbarButton>
+            <AddAttachmentButton onFilesAdded={handleAddFiles} />
+            <ToolbarSeparator />
+            <ToolbarButton
+              aria-label={localize('TEXT_INPUT_SEND_BUTTON_ALT')}
+              disabled={isMessageLengthExceeded}
```

Comment:
No `disabled`, use `aria-disabled` and `tabIndex={-1}`.

Reply:
Made this to be the implementation detail of the `disabled` prop